10_datastructures_lab/01.py

```diff
@@ -1,10 +1,19 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-"""
+import sys
+names=sys.argv[1:]
+print "names=", names
+hostsFile = "C:\\Python Course\\2\\hosts.txt"
+hostsDict = {}
+with open(hostsFile, 'r') as f:
+    for line in f:

```

Comment:
I prefer the more simple approach:
`host, ip = line.rstrip().split('=')`

You get both variables, and rstrip removes the newline character from the end